src/main/java/com/ita/if103java/ims/dto/WarehouseDto.java

```diff
@@ -42,11 +42,13 @@
 
     private boolean active;
 
+    private AddressDto addressDto;
+
     public WarehouseDto() {
     }
 
     public WarehouseDto(Long id, String name, String info, Integer capacity, boolean isBottom, Long parentID, Long accountID,
-                        Long topWarehouseID, boolean active) {
+                        Long topWarehouseID, boolean active, AddressDto addressDto) {
```

Comment:
How this new field is populated? Please add corresponding changes to the warehouse service